core/process/data.loader.php

```diff
@@ -426,10 +426,10 @@
 		// get all mythic pokemon
 		$req 		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon
 				   WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
-				   ORDER BY disappear_time DESC LIMIT 0,12";
+				   ORDER BY last_modified DESC LIMIT 0,12";
```

Comment:
So we just add it in the requested variables? That's how I understand it from the link.